.versions

```diff
@@ -1,89 +1,64 @@
-accounts-base@2.2.0
-aldeed:collection2@3.0.6
-allow-deny@1.1.0
-babel-compiler@7.7.0
-babel-runtime@1.5.0
+accounts-base@2.2.9
+aldeed:collection2@4.0.0-beta.7
```

Comment:
The latest collection2 version is [about to be](https://github.com/Meteor-Community-Packages/meteor-collection2/pull/449) 4.0.2 ;)